src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -322,17 +346,6 @@ ConvertToDistributedTable(Oid relationId, char *distributionColumnName,
 								  "foreign tables.")));
 	}
 
-	/* check that the relation does not contain any rows */
```

Comment:
Is removing this entirely safe? Can't users still reach this path calling `create_distributed_table` or `master_create_distributed_table` with non-hash distribution types? Previously they'd get this error, now they'll have no error, right?